### PR TITLE
develop: fix paging sql with multiple columns for cluster index table

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11341
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-paginate-results.md [hunk 1]
```diff
@@ -303,6 +303,10 @@ GROUP BY page_num
 ORDER BY page_num;
 ```
 
+> **Note:**
+>
+> The SQL will be executed as `TableFullScan`, when the data volume is large, the query will be slow, here you can [use TiFlash](/tiflash/tiflash-overview.md#use -tiflash) to speed up.
```
Comment (Oreoxmt) [inline develop/dev-guide-paginate-results.md:308]:
```suggestion
> The preceding SQL statement is executed as `TableFullScan`. When the data volume is large, the query will be slow, and you can [use TiFlash](/tiflash/tiflash-overview.md#use -tiflash) to speed up it.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by Mini256** [comment]
@Oreoxmt, thanks for your review!

File: develop/dev-guide-paginate-results.md [hunk 2]
```diff
@@ -303,6 +303,10 @@ GROUP BY page_num
 ORDER BY page_num;
 ```
 
+> **Note:**
+>
+> The preceding SQL statement is executed as `TableFullScan`. When the data volume is large, the query will be slow, and you can [use TiFlash](/tiflash/tiflash-overview.md#use -tiflash) to speed up it.
```
Comment (Icemap) [inline develop/dev-guide-paginate-results.md:308]:
```suggestion
> The preceding SQL statement is executed as `TableFullScan`. When the data volume is large, the query will be slow, and you can [use TiFlash](/tiflash/tiflash-overview.md#use-tiflash) to speed up it.
```

**Review by Icemap — APPROVED** [review]
Rest LGTM

**Review by winoros — APPROVED** [review]

**Pull request merged**